stations.csv

```diff
@@ -26092,8 +26092,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 28226;Sisak;sisak;7872901;;45.491997;16.373141;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800014;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HRADK;f;f;;;;;;;;;;;Sziszek;シサク;시사크;;;Сисак;;;錫薩克
 28227;Šibenik;sibenik;7876760;;43.730039;15.898106;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800017;f;;f;;f;6498;t;;f;;f;;f;;;;f;;f;;f;;f;HRADN;f;f;;;;;;Sebenico;;;;;;シベニク;시베니크;Szybenik;;Шибеник;;;希贝尼克
 28228;Kotoriba;kotoriba;7874808;;46.353613;16.808461;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800057;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HRADX;f;f;;;;;;;;;;;;;;;;;;;
-28229;Opatija-Matulji;opatija-matulji;7875611;;45.364809;14.321775;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800002;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HROPA;f;f;;;;;;;;;;;;;;;;;;;
-28230;Pula;pula;7877417;;44.879706;13.847199;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800003;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HRPUY;f;f;;;;;;Pola;;;;;;プーラ;풀라;;;Пула;;;普拉
+28229;Opatija-Matulji;opatija-matulji;7875611;;45.364809;14.321775;29610;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800002;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;HROPA;f;f;;;;;;;;;;;;;;;;;;;
+28230;Pula;pula;7877417;;44.879706;13.847199;;HR;Europe/Zagreb;f;f;f;f;f;f;;;f;;f;7800003;f;;f;;f;8988;t;;f;;f;;f;;;;f;;f;;f;;f;HRPUY;f;f;;;;;;Pola;;;;;;プーラ;풀라;;;Пула;;;普拉
```

Comment:
What is that added id? Why not in the same place as the others?

Reply:
Their is a new station: Opatija and I have updated Opatija-Matulji to set it as a child.